src/config.ts

```diff
@@ -21,6 +21,16 @@ export class Config {
    */
   static httpsPort: string = process.env.IR_HTTPS_PORT || '9443';
 
+  /**
```

Comment:
I don't think we need these config settings, in my opinion:
- It can be enabled by default, there's no real risk or issue when this is enabled.
- I don't see the need to configure the ping url (webservice URL's are also fixed for example)

There are already have quite some config options, so i'd rather minimize them a bit so it won't get too complex.

Reply:
Given the information that is exposed, I could imagine not everyone might feel comfortable having that in a predictable location. This is what is returned:

```
{
  "timestamp": 1521725807561,
  "uptime": 65.174,
  "application": {
    "name": "elvis-image-recognition",
    "version": "2.1.0",
    "pid": 17787,
    "title": "/usr/bin/node",
    "argv": [
      "node",
      "/opt/elvis-image-recognition/src/server.ts"
    ],
    "versions": {
      "http_parser": "2.7.1",
      "node": "6.12.3",
      "v8": "5.1.281.111",
      "uv": "1.10.2",
      "zlib": "1.2.7",
      "ares": "1.10.1-DEV",
      "icu": "50.1.2",
      "modules": "48",
      "openssl": "1.0.2k-fips"
    },
    "dependencies": {
      "@google-cloud/translate": "^1.0.0",
      "@google-cloud/vision": "^0.14.0",
      "@types/express": "^4.0.37",
      "@types/node": "^8.0.28",
      "aws-sdk": "^2.114.0",
      "bluebird": "^3.5.0",
      "body-parser": "^1.18.1",
      "clarifai": "^2.4.0",
      "console-stamp": "^0.2.5",
      "express": "^4.15.4",
      "express-ping": "^1.4.0",
      "limiter": "^1.1.2",
      "logger-request": "^3.8.0",
      "request": "^2.81.0",
      "secure-compare": "^3.0.1",
      "ts-node": "^3.3.0",
      "tslib": "~1.7.1",
      "uuid": "^3.0.1"
    }
  },
  "resources": {
    "memory": {
      "rss": 82272256,
      "heapTotal": 59572224,
      "heapUsed": 52425624,
      "external": 71110
    },
    "loadavg": [
      0.0029296875,
      0.0146484375,
      0.04541015625
    ],
    "cpu": [
      {
        "model": "Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz",
        "speed": 2400,
        "times": {
          "user": 1541400,
          "nice": 24700,
          "sys": 503900,
          "idle": 811204600,
          "irq": 0
        }
      },
      {
        "model": "Intel(R) Xeon(R) CPU E5-2676 v3 @ 2.40GHz",
        "speed": 2400,
        "times": {
          "user": 1830100,
          "nice": 28400,
          "sys": 996000,
          "idle": 810327600,
          "irq": 0
        }
      }
    ],
    "disk": [
      {
        "filesystem": "/dev/xvda1",
        "size": 41931756,
        "used": 2143212,
        "available": 39788544,
        "capacity": 0.06,
        "mount": "/"
      },
      {
        "filesystem": "devtmpfs",
        "size": 1919396,
        "used": 0,
        "available": 1919396,
        "capacity": 0,
        "mount": "/dev"
      },
      {
        "filesystem": "tmpfs",
        "size": 1940112,
        "used": 0,
        "available": 1940112,
        "capacity": 0,
        "mount": "/dev/shm"
      },
      {
        "filesystem": "tmpfs",
        "size": 1940112,
        "used": 16648,
        "available": 1923464,
        "capacity": 0.01,
        "mount": "/run"
      },
      {
        "filesystem": "tmpfs",
        "size": 1940112,
        "used": 0,
        "available": 1940112,
        "capacity": 0,
        "mount": "/sys/fs/cgroup"
      },
      {
        "filesystem": "tmpfs",
        "size": 388024,
        "used": 0,
        "available": 388024,
        "capacity": 0,
        "mount": "/run/user/1015"
      }
    ],
    "nics": {
      "lo": [
        {
          "address": "127.0.0.1",
          "netmask": "255.0.0.0",
          "family": "IPv4",
          "mac": "00:00:00:00:00:00",
          "internal": true
        },
        {
          "address": "::1",
          "netmask": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff",
          "family": "IPv6",
          "mac": "00:00:00:00:00:00",
          "scopeid": 0,
          "internal": true
        }
      ],
      "eth0": [
        {
          "address": "10.0.20.4",
          "netmask": "255.255.255.192",
          "family": "IPv4",
          "mac": "06:9d:4c:3a:aa:7e",
          "internal": false
        },
        {
          "address": "fe80::49d:4cff:fe3a:aa7e",
          "netmask": "ffff:ffff:ffff:ffff::",
          "family": "IPv6",
          "mac": "06:9d:4c:3a:aa:7e",
          "scopeid": 2,
          "internal": false
        }
      ]
    }
  },
  "system": {
    "arch": "x64",
    "platform": "linux",
    "type": "Linux",
    "release": "3.10.0-693.21.1.el7.x86_64",
    "hostname": "meh.",
    "uptime": 81354,
    "cores": 2,
    "memory": 3973349376
  }
}
```